ebuild-maintenance/general/text.xml

```diff
@@ -0,0 +1,530 @@
+available</uri> in Gentoo's infra.
+</p>
+
+<p>
+After committing, use <c>pkgcheck scan --commits</c> for a final check before
+issuing <c>git pull &amp;&amp; git push</c> or <c>git pull &amp;&amp; pkgdev
+push</c>. Of course, <c>git log -p</c> before pushing doesn't hurt either. <c>
+git status</c> will help you check that you don't have any modified, uncommited
+work (like a .patch file) before pushing.
+<uri link="https://blogs.gentoo.org/mgorny/2019/12/12/a-better-ebuild-workflow-with-pure-git-and-pkgcheck/">
+"A better ebuild workflow with pure git and pkgcheck"</uri> guide if you're not
+too familiar with git.
+</p>
+
+<p>
+Gentoo's
+<uri link="https://qa-reports.gentoo.org/output/gentoo-ci/output.html">CI
+system </uri> runs pkgcheck, so familiarizing yourself with pkgcheck and its
+warnings is encouraged. If you add a new warning to the tree, you will get an
+e-mail from "repomirrorci@g.o" with the warning, and this could most likely
+have been prevented by running <c>pkgcheck scan --commits</c> before pushing
+your changes. A new CI error will prevent metadata being generated to Gentoo's
+sync-repositories, and therefore deny users the latest updates until the CI
+error is fixed.
+</p>
+
+<p>
+By default <c>pkgcheck</c> enables a lot of checks. Some of them are more
+useful than the others. Pkgcheck can be configured via the <c>
+~/.config/pkgcheck/pkgcheck.conf</c> file. For example, a sane configuration
+could look like this:
+</p>
+
+<pre>
+# ~/.config/pkgcheck/pkgcheck.conf
+#
+# -ImlateCheck = PotentialStable, all of these checks are just noise.
+[DEFAULT]
+checks = -ImlateCheck,-RedundantVersionCheck,-UnstableOnlyCheck
+</pre>
+
+<p>
+Refer to the <uri link="https://pkgcore.GitHub.io/pkgcheck/man/pkgcheck.html">
+official documentation</uri> of the flags and edit to your needs.
+
+You can also just enable the checks you wish from the command line, e.g., <c>
+pkgcheck scan -c StableRequestCheck</c>. More on that below.
+</p>
+
+<p>
+Ask a fellow developer if you're unsure of the meaning or how to fix a warning.
+If in doubt about who to ask, head to the #gentoo-dev, #gentoo-qa, or
+#gentoo-dev-help (for non-developers too) IRC channels. You can, and you
+should, of course ask before pushing if something is not fully clear to you.
+</p>
+
+</body>
+
+<subsubsection>
+<title>GitHub pull requests</title>
+<body>
+
+<p>
+You may find yourself in a lucky position where enthusiastic users provide
+contributions to your packages via GitHub pull requests. Using GitHub is not
+mandatory and Bugzilla is still the main channel for reporting enhancements to
+your packages. But consider trying out GitHub, as it's getting more and more
+popular and we're seeing more and more contributions pushed there. Since a
+distribution would be nothing without its users, we shouldn't ignore our
+users in popular platforms. Chances are, they end up becoming official
```

Comment:
I see this section very critical. As a free software distro, we must not advertise a proprietary and [arguably unethical](https://www.gnu.org/software/repo-criteria-evaluation.html) Microsoft platform in our official documention.

Mentioning Github is o.k., but phrases like "you may find yourself in a lucky position" or "consider trying out GitHub, as it's getting more and more popular" definitely don't belong here. Indirectly, this would also put pressure on developers on using Github, while we have clarified multiple times (and our Social Contract mandates) that its use in entirely optional.

Reply:
```
You may find yourself in a position where users provide contributions to your 
packages via GitHub pull requests. Gentoo is seeing more and more contributions 
pushed there, as GitHub is getting more popular. That being said, using GitHub 
is not mandatory and Bugzilla is still the main channel for reporting 
enhancements to your packages.
```

Reply:
WFM, if you word the second sentence in a more neutral way. Something along the lines of:
"As GitHub is a popular platform, Gentoo is seeing many contributions pushed there."